modules/widgets/flickr/form.php

```diff
@@ -51,6 +51,16 @@ class="widefat"
 	</select>
 </p>
 
+<p>
+	<label>
+		<?php esc_html_e( 'Open images in new tab?', 'jetpack' ); ?>
+	</label>
+	<input
```

Comment:
I would place the checkbox before the label, I think it feels more natural.

Reply:
Good call. Actually reviewing some of the other checkboxes in other Jetpack widgets it looks like the checkbox and label text is nested within label tags. Made the change for this to be in-line with this pattern.